lib/nfs_common.pm

```diff
@@ -219,8 +219,13 @@ sub config_service {
     send_key 'alt-s';
 
     # Disable NFSv4
-    send_key 'alt-v';
-    wait_still_screen 1;
+    do {
+        assert_screen([qw(nfsv4-disabled nfsv4-enabled)], 120);
+        if (match_has_tag('nfsv4-enabled')) {
+            send_key 'alt-v';
+            wait_still_screen 1;
+        }
+    } while (not match_has_tag('nfsv4-disabled'));
```

Comment:
Why need the loop? I think multi-tag is enough.

Reply:
will try if it works. thanks

Reply:
works and changed the code https://openqa.nue.suse.com/tests/5961025#step/install_service/43